deploy/provision.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd -P -- "$(dirname -- "$0")/terraform" || exit
+
+echo "WARNING:"
+echo "This will create a new EC2 instance. Do not use this script if you are trying to deploy new code."
+echo
+
+terraform fmt
+terraform validate
```

Comment:
Is there any output from here? What happens if we have invalid terraform, will it just try to init & apply anyway?

Reply:
validation is also done by `terraform apply`, so I will remove this